src/fal/planner/plan.py

```diff
@@ -28,8 +29,17 @@ def from_execution_plan(
     ) -> FilteredGraph:
         graph = origin_graph.copy_graph()
         for node in origin_graph.graph.nodes:
-            if node not in execution_plan.dbt_models:
+            if (
+                node
+                not in execution_plan.dbt_models
+                + execution_plan.before_scripts
+                + execution_plan.after_scripts
+            ):
                 graph.remove_node(node)
+
+        if execution_plan.after_scripts:
+            logger.warn("Using after scripts is now deprecated. Please consider migrating to post-hooks.")
```

Comment:
Just to confirm: before scripts will still stick around right? @chamini2 @turbo1912

Reply:
For now at least, this looks ok. We will have to decide later if pre-hooks will be a thing.